app/renderer/components/Inspector/HighlighterRects.js

```diff
@@ -83,7 +83,7 @@ export default class HighlighterRects extends Component {
         path: source.path,
         keyCode: null,
         container: false,
-        accessible: source.attributes.accessible
+        accessible: source.attributes ? source.attributes.accessible : null
```

Comment:
you could probably use `?.` operator

Reply:
i tried that, the build system the inspector uses doesn't handle it